java/flight/flight-core/src/test/java/org/apache/arrow/flight/perf/PerformanceTestServer.java

```diff
@@ -68,7 +68,7 @@ public void register(FlightProducer.ServerStreamListener listener) {
 
       @Override
       public WaitResult waitForListener(long timeout) {
-        while (!listener.isReady()) {
```

Comment:
this is unrelated to the change is suppose?

Reply:
This was needed or the test would get stuck - I think it's just I bumped something that exposed a race condition.